source/Process.Application/ProcessInitializationHandlers/InitializeMeteredDataForMeasurementPointHandler.cs

```diff
@@ -35,11 +47,47 @@ public bool CanHandle(string processTypeToInitialize)
         return processTypeToInitialize.Equals(nameof(InitializeMeteredDataForMeasurementPointMessageProcessDto), StringComparison.Ordinal);
     }
 
-    public Task ProcessAsync(byte[] processInitializationData)
+    public async Task ProcessAsync(byte[] processInitializationData)
     {
-        var marketMessage = _serializer.Deserialize<InitializeMeteredDataForMeasurementPointMessageProcessDto>(System.Text.Encoding.UTF8.GetString(processInitializationData));
+        var marketMessage =
+            _serializer.Deserialize<InitializeMeteredDataForMeasurementPointMessageProcessDto>(
+                System.Text.Encoding.UTF8.GetString(processInitializationData));
         _logger.LogInformation("Received InitializeAggregatedMeasureDataProcess for message {MessageId}", marketMessage.MessageId);
-        // Nothing to see here yet.
-        return Task.CompletedTask;
+
+        foreach (var series in marketMessage.Series)
+        {
+            await _outgoingMessagesClient.EnqueueAndCommitAsync(
+                    new MeteredDataForMeasurementPointMessageProcessDto(
+                        EventId.From(marketMessage.MessageId),
+                        new Actor(series.RequestedByActor.ActorNumber, series.RequestedByActor.ActorRole),
```

Comment:
I suggest we use a fictitious energy supplier instead. It aligns better with the business's expectations for the demo.

```suggestion
                        new Actor("5790000392551", ActorRole.EnergySupplier),
```